joplin/pages/news_page/models.py

```diff
@@ -0,0 +1,81 @@
+from django.db import models
+
+from wagtail.core.fields import RichTextField
+from wagtail.admin.edit_handlers import FieldPanel, PageChooserPanel
+from wagtail.snippets.edit_handlers import SnippetChooserPanel
+
+from base.forms import ServicePageForm
+
+from pages.base_page.models import JanisBasePage
+from pages.department_page.models import DepartmentPage
+from snippets.contact.models import Contact
+
+from base.models.widgets import countMe
+
+from publish_preflight.requirements import FieldPublishRequirement, DepartmentPublishRequirement
+
+from publish_preflight.forms import PublishPreflightForm
+
+WYSIWYG_NEWS_BODY = ['ul', 'ol', 'link']
+
+
+class NewsPageForm(PublishPreflightForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+
+class NewsPage(JanisBasePage):
+    janis_url_page_type = "news"
+
+    body = RichTextField(
+        features=WYSIWYG_NEWS_BODY,
+        verbose_name='Body',
+        blank=True
+    )
+
+    base_form_class = ServicePageForm
+
+    contact = models.ForeignKey(Contact, blank=True, null=True, on_delete=models.SET_NULL)
+
+    written_for_department = models.ForeignKey(DepartmentPage, blank=True, null=True, on_delete=models.SET_NULL,
+                                               verbose_name="Assign a different department",
+                                               help_text="If this news is written for another department, select that department below. This ensures that this news is associated with that department's news content")
+
+    publish_requirements = (
+        FieldPublishRequirement("body", message="🤠 can't publish without a body 🌵", langs=["en"]),
+        FieldPublishRequirement("contact", message="🤠 can't publish without a contact 🌵"),
+        DepartmentPublishRequirement(message="🤠 can't publish without a department, this should only happen to admins that forgot to pick a department in the modal, or users that aren't in a department group. This requirement will not be satisfied by the 'different department' field 🌵"),
+    )
+
+    content_panels = [
+        # todo: we want these to say headline, not title
```

Comment:
verbose_name?

Reply:
can we do verbose_name on the content panels? I think it'd need to happen in the Field definition and title's field is all the way back on wagtail's page model